examples/passport-login/index.ts

```diff
@@ -0,0 +1 @@
+export * from './src';
```

Comment:
Copy right header is missing. Please run `../../packages/cli/bin/cli-main.js copyright` in `examples/passport-login` to fix it.

Reply:
agreed, the command `../../packages/cli/bin/cli-main.js copyright` is run and all files have headers now. thank you.